### PR TITLE
Added Helio Workstation version 2.0 (3.7).

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24288)

### CODE REVIEW

File: manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Helio.HelioWorkstation
+PackageVersion: "3.7"
```
Comment (ItzLevvie) [inline manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml:3]:
```suggestion
PackageVersion: 2.0
```

File: manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+PackageName: Helio Workstation
```
Comment (ItzLevvie) [inline manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml:4]:
```suggestion
PackageName: Peter Rudenko
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml (hunk 1 above)
Comment (jedieaston) [reply manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml:3]: We'll have upgrade issues if we do this. I'm going to wait on these until I have time to fix the other ones.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**